### PR TITLE
Make the 3D stokes drift validation case even better

### PR DESCRIPTION
**Pull request by glwagner**

### CODE REVIEW

File: validation/stokes_drift/surface_wave_quasi_geostrophic_flow.jl [hunk 1]
```diff
@@ -98,18 +108,56 @@ n = Observable(1)
...
+xa = reshape(xa, Nx, 1)
+ya = reshape(ya, 1, Ny)
+
+An = @lift begin
+    t = times[$n]
+    ξ = @. xa - cᵍ * t
+    A.(ξ, ya)
+end
+
+xu = xu .* 1e-3
```
Comment (navidcy) [inline validation/stokes_drift/surface_wave_quasi_geostrophic_flow.jl:129]:
```suggestion
# convert m -> km
xu = xu .* 1e-3
```

File: validation/stokes_drift/surface_wave_quasi_geostrophic_flow.jl [hunk 2]
```diff
@@ -98,18 +108,56 @@ n = Observable(1)
...
+
+heatmap!(axu, xu, yu, un, colormap=:balance, colorrange=(-ulim, ulim))
+#heatmap!(axu, xe, ye, en, colormap=:solar, colorrange=(0, elim))
+contour!(axu, xc, yc, An, color=:gray, levels=5)
+
+heatmap!(axw, xw, yw, wn, colormap=:balance, colorrange=(-wlim, wlim))
+contour!(axw, xc, yc, An, color=:gray, levels=5)
 
 record(fig, "surface_wave_quasi_geostrophic_induced_flow.mp4", 1:Nt, framerate=8) do nn
+#record(fig, "surface_wave_non_rotating_induced_flow.mp4", 1:Nt, framerate=8) do nn
```
Comment (navidcy) [inline validation/stokes_drift/surface_wave_quasi_geostrophic_flow.jl:160]: why comment the movie?
```suggestion
#record(fig, "surface_wave_non_rotating_induced_flow.mp4", 1:Nt, framerate=8) do nn
```

Comment (navidcy) [reply validation/stokes_drift/surface_wave_quasi_geostrophic_flow.jl:160]:
```suggestion
record(fig, "surface_wave_non_rotating_induced_flow.mp4", 1:Nt, framerate=8) do nn
```

**Review by navidcy — APPROVED** [review]
lgtm!

(disclaimer: I didn't run the script)

**Comment by glwagner** [comment]
It just makes the script a little better. Here's some movies of an initial value problem corresponding to a impulsively generated wave packet...

non-rotating case (basically McIntyre / Bretherton wave-induced flow):

https://github.com/CliMA/Oceananigans.jl/assets/15271942/d3c74b72-245d-4fb4-9269-a7bc5392c6f4

rotating case (wave-induced flow is geostrophic, wave-transmitted flow is a near-inertial wave)

https://github.com/CliMA/Oceananigans.jl/assets/15271942/d2e697b5-27f5-4b24-bb37-7724b25416be

**Pull request merged**